.github/workflows/examples-efr32.yaml

```diff
@@ -15,9 +15,10 @@
 name: Build example - EFR32
 
 on:
-  push:
-    branches-ignore:
-      - 'dependabot/**'
+    push:
```

Comment:
Maybe the problem is with inconsistent indentation? The rest of this file is indented with 2 spaces, not 4.

Reply:
@arkq I would agree with you on this. I noticed that too, but given the file’s long-standing state, I hesitated to make the indentation changes with this commit.

Reply:
Yes, but YAML should be indented by 2 spaces (at least that's what most YAML linters do)